docs: add missing guides, examples, images to CODEOWNERS

gkalpak opened the pull request:
Some guides/examples/images from `aio/content/` did not have owners (in `.github/CODEOWNERS`) and fell back to the default owners.
This PR adds owners for all guides/examples/images.

gkalpak commented on .github/CODEOWNERS:
```diff
@@ -686,25 +714,46 @@ testing/**                                                      @angular/fw-test
+#  Docs: quick reference
+# ================================================
+
+# TODO: Who owns these?
```
cc @IgorMinar

IgorMinar replied:
cheatsheet should go under core.

glossary and public api should go into the public_api group at the bottom

gkalpak replied:
I assume you meant _"glossary and styleguide"_ :smiley:

jenniferfell replied:
I think of glossary and architecture docs as part of docs-intro, similar to getting started and tutorial. I suppose glossary could be public-api, but architecture is definitely similar to other intro materials.

IgorMinar replied:
I think that all three of these should go under public api as they are the most visible terminology definitions, recommendations and "style" guides.

gkalpak commented on .github/CODEOWNERS:
```diff
@@ -735,6 +794,20 @@ testing/**                                                      @angular/fw-test
+#  Docs: Hidden
+# ================================================
+
+# TODO: Who owns these?
```
cc @IgorMinar

IgorMinar replied:
let's move all of these into the docs-infra group

IgorMinar reviewed (commented):
responding to questions...

gkalpak commented on .github/CODEOWNERS:
```diff
@@ -715,12 +761,22 @@ testing/**                                                      @angular/fw-test
+# TODO: Do we need a new `Docs: libraries` category?
+/aio/content/guide/creating-libraries.md                        @angular/fw-docs-packaging @angular/framework-global-approvers @angular/framework-global-approvers-for-docs-only-changes
+/aio/content/guide/libraries.md                                 @angular/fw-docs-packaging @angular/framework-global-approvers @angular/framework-global-approvers-for-docs-only-changes
+/aio/content/guide/using-libraries.md                           @angular/fw-docs-packaging @angular/framework-global-approvers @angular/framework-global-approvers-for-docs-only-changes
```
@IgorMinar, what about these ones?

jenniferfell replied:
It would be good to have a codeowners group(s) that alexE and hans can approve, for libraries, schematics, CLI overview page.

IgorMinar replied:
libraries should likely be a separate category with a separate team owning it - the owners should be some folks on the tooling team and me. @alexeagle can give you the list of tooling team ppl

IgorMinar replied:
and yesk, I previously said that we should reuse ngtools group, but now I'm thinking that reviewing library docs changes requires a very specific knowledge, so a custom group would be better.

alexeagle replied:
Hmm, I think it's probably @hansl, @mgechev, @alan-agius4

gkalpak commented:
(Note: The CI failure in `test_ivy_aot` is inherited from master and that job's outcome is not affected by this PR.)

IgorMinar reviewed (approved):
the rest looks good.

please resolve comments and remove all TODOs and `???` and send for merge.

gkalpak committed:
docs: add missing guides, examples, images to CODEOWNERS

gkalpak committed:
test(docs-infra): add script for verifying all guides/examples/images have owners

It is useful for manually checking that all guides/examples/images have
owners in `.github/CODEOWNERS`, but is not used for automatic
verification (e.g. on CI) for now.

gkalpak committed:
docs: remove unused examples and images

gkalpak commented:
I made all necessary changes. @IgorMinar, you need to create the [fw-docs-libraries][1] GitHub group.

[1]: https://github.com/angular/angular/pull/28597/files#diff-5cfb934ca0ec4c1743905a51dc428b84R298

gkalpak commented:
I found out I can create the GitHub team myself, so I created [tools-docs-libraries][1].

[1]: https://github.com/orgs/angular/teams/tools-docs-libraries

gkalpak commented:
merge-assistance: @IgorMinar said I can send for merge (https://github.com/angular/angular/pull/28597#pullrequestreview-208955511).

kara commented:
Note: The changes aren't applying cleanly to patch, so merging it into master now, and George has agreed to create another PR against the patch branch the resolves the conflicts.
